yandexcloud/_operation_waiter.py

```diff
@@ -1,14 +1,73 @@
 import time
+from datetime import datetime
+import logging
 
 from yandex.cloud.operation.operation_service_pb2_grpc import OperationServiceStub
 from yandex.cloud.operation.operation_service_pb2 import GetOperationRequest
+from yandexcloud.operations import OperationResult, OperationError
 
 
 def operation_waiter(sdk, operation_id, timeout):
     operation_service = sdk.client(OperationServiceStub)
     return OperationWaiter(operation_id, operation_service, timeout)
 
 
+def wait_for_operation(sdk, operation_id, timeout, print_to_stream=None):
+    waiter = operation_waiter(sdk, operation_id, timeout)
+    for _ in waiter:
+        if print_to_stream:
+            print_to_stream.write('.')
+            print_to_stream.flush()
+        time.sleep(1)
+    if print_to_stream:
+        print_to_stream.write('done')
+    return waiter.operation
+
+
+def get_operation_result(sdk, operation, response_type=None, meta_type=None, timeout=None, logger=None):
+    if not logger:
+        logger = logging
```

Comment:
I'd rather use null logger that doesn't log anything.

Reply:
Done